docs/development/ksd-product-roadmap.md

```diff
@@ -0,0 +1,35 @@
+---
+title: KSD Product Roadmap
+---
+
+# KSD Product Roadmap
+
+## Big ideas - subject to change
+
+The Koor Storage Distribution includes features that work on top of Rook and Ceph to enhance the experience of storage professionals. Here is a look at the features in KSD today and ideas for the new features and improvement in the future. Although this represents our best intentions at the time of publishing, everything here is subject to change.
+
+## Feature areas
+
+### Koor Operator
+
+| Feature   |   State   | Description                     |
+| --------- |   :---:   | ------------------------------- |
+| something | [0.2.0 released](https://github.com/koor-tech/koor-operator/releases/tag/koor-operator-0.2.0) | describe something it does already |
+| version update notifications | in development |  a brief statement of what this will do |
+| automated Ceph upgrades | future | a brief statement of what this will do |
+
+
+### Security
+
+| Feature   |   State   | Description                     |
+| --------- |   :---:   | ------------------------------- |
+| SSO Setup | released | Instructions for setting up SSO |
+
+
+### Troubleshooting tools
+
+| Feature   |   State   | Description                     |
+| --------- |   :---:   | ------------------------------- |
+| Extended Ceph Exporter | [1.2.8 released](https://github.com/koor-tech/extended-ceph-exporter/releases/tag/extended-ceph-exporter-1.2.8) | Prometheus exporter to provide "extended" metrics about a Ceph cluster's running components |
+| Gather Info | internal | Scripts to collect Ceph operational data to understand performance characteristics of the system |
```

Comment:
```suggestion
| Gather Info | released | Scripts to collect Ceph operational data to understand performance characteristics of the system |
```

Reply:
Now I understand what you were saying earlier. Perhaps the state of 'public' is more accurate than 'released', since the repo shows 'No releases published.'